apps/whale-api/src/module.api/poolpair.service.ts

```diff
@@ -70,32 +72,31 @@ export class PoolPairService {
   }
 
   /**
-   * TODO(fuxingloh): graph based matrix resolution
-   * Currently implemented with fix pair derivation
-   * Ideally should use vertex directed graph where we can always find total liquidity if it can be resolved.
+   * Graph based matrix resolution
+   * Returns the total liquidity of the poolpair by finding its best path to USDT
```

Comment:
I'm worried about the performance.

But I assume this method is cached, shouldn't be much of a hit on performance.